Static/DataSource.swift

```diff
@@ -277,4 +277,18 @@ extension DataSource: UITableViewDelegate {
             row.accessory.selection?()
         }
     }
+    
+    public func tableView(tableView: UITableView, shouldShowMenuForRowAtIndexPath indexPath: NSIndexPath) -> Bool {
+        return rowForIndexPath(indexPath)?.canCopy ?? false
+    }
+    
+    public func tableView(tableView: UITableView, canPerformAction action: Selector, forRowAtIndexPath indexPath: NSIndexPath, withSender sender: AnyObject?) -> Bool {
+        return action == #selector(NSObject.copy(_:)) && (rowForIndexPath(indexPath)?.canCopy ?? false)
```

Comment:
What's the rationale here for only supporting `#selector(NSObject.copy(_:)`? Could we have supported other actions?